modules/SitecoreImageBuilder/SitecoreImageBuilder.psm1

```diff
@@ -745,8 +760,14 @@ function Get-WindowsServerCoreToNanoServerVersionMap
 
 function Get-LatestSupportedVersion
 {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [array]$Specs
+    )
+
     # load Windows image specifications
-    $specs = Get-BuildSpecifications -Path (Join-Path $PSScriptRoot "\..\..\windows")
+    #$specs = Get-BuildSpecifications -Path (Join-Path $PSScriptRoot "\..\..\windows")
```

Comment:
if/when certain this is no longer required please remove commented code.

Reply:
I removed this.